node_modules/oae-admin/manageuser/css/manageuser.css

```diff
@@ -57,8 +57,13 @@
 
 /* ACTIONS */
 
-#manageuser-modal #manageuser-become-user {
-    margin-bottom: 5px;
+#manageuser-modal .checkbox {
```

Comment:
This should be bellow the `#manageuser-become-user` and `#manageuser-delete-user` rules as the checkbox appears bellow them in the UI. We try to maintain the same flow in the CSS as in the UI itself